tests/pytests/test_vecsim.py

```diff
@@ -1685,7 +1691,7 @@ def __init__(self):
             raise SkipTest()
         self.env = Env(moduleArgs='DEFAULT_DIALECT 2 ON_TIMEOUT FAIL')
         n_shards = self.env.shardsCount
-        self.index_sizes = {'FLAT': 80000 * n_shards, 'HNSW': 10000 * n_shards}
+        self.index_sizes = {'FLAT': 80000 * n_shards, 'HNSW': 10000 * n_shards, 'SVS-VAMANA': 10000 * n_shards}
         self.hybrid_modes = ['BATCHES', 'ADHOC_BF']
         self.dim = 10
         self.type = 'FLOAT64'
```

Comment:
For better code, this could be changed to a type that SVS-VAMANA can support (FLOAT32, for example), so that svs index will use `self.type` as well.